Tests/helper.py

```diff
@@ -15,6 +15,9 @@
 
 from PIL import Image, ImageMath, features
 
+# overridden in conftest.py
+pytestconfig = {"option": {"verbose": 0}}
```

Comment:
Since it's overridden, these lines don't need to be here?

Reply:
Probably. I wasn't sure if that test code might ever be run without a pytest config having been loaded first though.